lib/src/inline_parser.dart

```diff
@@ -1103,10 +1152,12 @@ class LinkSyntax extends DelimiterSyntax {
       // Normally, label text does not get parsed as inline Markdown. However,
       // for the benefit of the link resolver, we need to at least escape
       // brackets, so that, e.g. a link resolver can receive `[\[\]]` as `[]`.
-      final resolved = linkResolver(label
-          .replaceAll(r'\\', r'\')
-          .replaceAll(r'\[', '[')
-          .replaceAll(r'\]', ']'));
+      final resolved = linkResolver(
```

Comment:
I think this should be reverted. It just feels very strange to me in cases like this where there is only one parameter...